SplunkRumWorkspace/SplunkRum/SplunkRum/Session.swift

```diff
@@ -17,12 +17,14 @@ limitations under the License.
 
 import Foundation
 
-let MAX_SESSION_AGE_SECONDS = 4 * 60 * 60
+let MAX_SESSION_AGE_SECONDS = 40   // 4 * 60 * 60
```

Comment:
Think this is left in from debugging (40s)

Reply:
changed.